tests/integration/psql_import/test_import_psql.py

```diff
@@ -287,3 +287,89 @@ def test_start_up(docker_compose):
             return_results=True,
         )
         assert len(results[0].docs[0].matches) == len(docs)
+
+
+@pytest.mark.parametrize('shards', [1])
+@pytest.mark.parametrize('nr_docs', [100])
+@pytest.mark.parametrize('emb_size', [10])
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
+def test_psql_sync_delta(
+    tmpdir,
+    nr_docs,
+    emb_size,
+    shards,
+    docker_compose,
+):
+    # for psql to start
+    time.sleep(2)
+
+    top_k = 5
+    batch_size = min(1000, nr_docs)
+    docs_original = get_batch_iterator(
+        batches=nr_docs // batch_size, batch_size=batch_size, emb_size=emb_size
+    )
+    docs_original = DocumentArray(flatten(docs_original))
+    assert len(docs_original) == nr_docs
```

Comment:
Let's simplify - you only need `get_documents`, and no need to test jina core functionality